src/System.Collections.NonGeneric/tests/Hashtable/AddStressTests.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections;
+using Xunit;
+
+namespace System.Collections.HashtableTests
+{
+    public partial class AddStressTests
+    {
+        [Fact]
+        public void TestAddRemoveLargeAmountNumbers()
+        {
```

Comment:
How long does this take to run?  Should it be declared as an [OuterLoop] test case?